packages/containerd-2.1/containerd-2.1.spec

```diff
@@ -38,13 +39,13 @@ Source1000: clarify.toml
 
 # Patch to support moving from containerd-1.7 to 2.x
 Patch1001: 1001-Revert-Don-t-allow-io_uring-related-syscalls-in-the-.patch
-Patch1002: 1002-Ensure-errContentRangeIgnored-error-when-range-get-r.patch
 
 BuildRequires: git
 BuildRequires: %{_cross_os}glibc-devel
 Requires: %{_cross_os}runc
 Requires: %{name}(optimized-gunzip)
 Requires: %{name}(binaries)
+Requires: %{_cross_os}notation-image-verifier
```

Comment:
If the only settings plugin that will include these new settings is `aws-ecs-3`, then we should omit this and just add it at the variant level. If you're planning to include everywhere containerd 2.1 is included, then this is fine.

Maybe these new files should go into the `notation-image-verifier` package? It doesn't seem like `containerd-2.1` has to do anything special.